sectionsUtilityTool/src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -91,6 +91,13 @@ private void canvasRestApiCall(HttpServletRequest request,
 		 }else {
 			url=canvasURL+pathInfo;
 		}
+		String sessionId = request.getSession().getId();
+		StringBuilder strgBlder =new StringBuilder();
```

Comment:
I forgot to mention earlier that variable names should be informative.  They should describe their purpose or contents.  A better name for "strgBlder" might be "sessionInfo".  It's good to use long names, too.  Bytes are cheap and many editors will autocomplete to save you the typing.